src/modules.c

```diff
@@ -155,9 +156,10 @@ Int ActivateModule(StructInitInfo * info)
         if (info->initLibrary) {
             // Start a new executor to run the outer function of the module in
             // global context
-            ExecBegin(STATE(BottomLVars));
+            Bag oldLvars = STATE(CurrLVars);
+            SWITCH_TO_OLD_LVARS(STATE(BottomLVars));
             res = res || info->initLibrary(info);
-            ExecEnd(res);
+            SWITCH_TO_OLD_LVARS(oldLvars);
```

Comment:
Note that I am not even sure we need to save&restore `CurrLVars` here; but historically we do. But I really wonder what a situation would be were we need to do this, yet don't need to save&restore some of the other global state...?